nibabel/cifti2/cifti2.py

```diff
@@ -0,0 +1,848 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+''' Read / write access to NIfTI2 image format
+
+Format described here:
+
+    http://www.nitrc.org/forum/message.php?msg_id=3738
+
+Stuff about the CIFTI2 file format here:
+
+    http://www.nitrc.org/plugins/mwiki/index.php/cifti2:ConnectivityMatrixFileFormats
+
+'''
+from __future__ import division, print_function, absolute_import
+
+import re
```

Comment:
Additionally, the nitrc wiki link to "cifti2:ConnectivityMatrixFileFormats" is incorrect.  This was probably a find/replace error, but the corresponding "cifti:ConnectivityMatrixFileFormats" wiki page is only about cifti1, which I'm guessing you aren't supporting (because of backwards dims).